WizardPanels/useFocusOnTransition.js

```diff
@@ -28,27 +30,7 @@ const transitionHandlers = {
 
 function useFocusOnTransition (config) {
 	const {current} = useRef({timerId: null, index: config.index});
-	const [complete, setTransition] = useState(false);
-	const handlers = useHandlers(transitionHandlers, config, {setTransition, current});
-	const focusIndex = current.index !== config.index && config.noAnimation ? config.index : -1;
-
-	useEffect(() => {
-		if (complete) {
-			// FIXME: onTransition fires while the departing view still exists so focusing the Panel
-			// will generally result in focusing a component in the departing view because it is
-			// first in DOM order.
-			//
-			// Deferring a tick allows that view to be removed but this is not an ideal solution.
-			current.timerId = setTimeout(() => {
-				const currentSpotlight = Spotlight.getCurrent();
-				if (config.spotlightId && !currentSpotlight) {
-					Spotlight.focus(config.spotlightId);
-				}
-			}, 16);
-		}
-
-		return () => clearTimeout(current.timerId);
-	}, [complete, config.spotlightId, focusIndex, current]);
```

Comment:
Let's test without these dependencies on the board.

Reply:
If we remove the dependecy, this use Effect called whenever WizardPanelsRouter called.
It is even called when a child is updated. please refer the "WithChangingChildren" sample